eZ/Publish/Core/Search/Legacy/Tests/Content/HandlerContentTest.php

```diff
@@ -1436,4 +1437,11 @@ public function testFieldRelationFilterInArrayNotMatch()
             )
         );
     }
+
+    public function testGetNonExistingFieldDefinition()
+    {
+        $this->expectException(\eZ\Publish\Core\Base\Exceptions\NotFoundException::class);
```

Comment:
Please rely on API NotFoundException instead and import it keeping here only:
```suggestion
        $this->expectException(NotFoundException::class);
```